frontend/public/locales/en/translation.json

```
@@ -378,7 +378,7 @@
     "privacyPolicy": "Privacy Policy",
     "whatPersonalDataTournesolCollectAndWhy": "What personal data Tournesol collects and why?",
     "ratings": "Ratings",
-    "whyTournesolCollectsRatings": "Tournesol is a tool for collaborative decisions. This tool requires to elicit, infer and aggregate contributors' judgments on the quality of multiple types of alternatives. These alternatives can for example be online videos or candidates to a presidential election. We want to make sure that the users of the plateform are well aware that political opinions are sensitive personnal information and that Tournesol collects and uses the data provided and saved on the plateform. To do so, Tournesol collects the data provided by the contributors when they compare pairs of content.",
+    "whyTournesolCollectsRatings": "Tournesol is a tool for collaborative decisions. This tool requires to elicit, infer and aggregate contributors' judgments on the quality of multiple types of alternatives. These alternatives can for example be online videos or candidates to a presidential election. We want to make sure that the users of the platform are well aware that political opinions are sensitive personnel information and that Tournesol collects and uses the data provided and saved on the platform. To do so, Tournesol collects the data provided by the contributors when they compare pairs of content.",
```

I think we want the adjective personal here, not the personal noun.

```suggestion
    "whyTournesolCollectsRatings": "Tournesol is a tool for collaborative decisions. This tool requires to elicit, infer and aggregate contributors' judgments on the quality of multiple types of alternatives. These alternatives can for example be online videos or candidates to a presidential election. We want to make sure that the users of the platform are well aware that political opinions are sensitive personal information and that Tournesol collects and uses the data provided and saved on the platform. To do so, Tournesol collects the data provided by the contributors when they compare pairs of content.",
```